src/Compilers/CSharp/Portable/FlowAnalysis/EmptyStructTypeCache.cs

```diff
@@ -166,6 +166,14 @@ private FieldSymbol GetActualInstanceField(Symbol member, NamedTypeSymbol type)
                 switch (member.Kind)
                 {
                     case SymbolKind.Field:
+                        // Do not report virtual tuple fields.
+                        // They are additional aliases to the fields of the underlying struct or nested extensions.
+                        // and as such are already accounted for via the nonvirtual fields.
+                        if (field is TupleVirtualElementFieldSymbol)
```

Comment:
Added a virtual property for the virtual field check.

---
In reply to: [74851677](https://github.com/dotnet/roslyn/pull/13107#discussion_r74851677) [](ancestors = 74851677,74525116,74524647)